src/main/java/com/yahoo/bullet/parsing/CastExpression.java

```diff
@@ -0,0 +1,60 @@
+/*
+ *  Copyright 2018, Oath Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.parsing;
+
+import com.google.gson.annotations.Expose;
+import com.yahoo.bullet.common.BulletError;
+import com.yahoo.bullet.typesystem.Type;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static com.yahoo.bullet.common.BulletError.makeError;
+
+@Getter @Setter
+public class CastExpression extends Expression {
+    public static final BulletError CAST_EXPRESSION_REQUIRES_VALID_EXPRESSION_ERROR =
+            makeError("Casting needs a valid expression field", "Please add a valid expression.");
+    public static final BulletError CAST_EXPRESSION_REQUIRES_PRIMITIVE_TYPE_ERROR =
```

Comment:
I guess my above comment applies here and a few other places too.  Static errors are appealing from a coding perspective, but from a user's perspective, being able to print the actual expression so the user knows where to look might be preferable.